src/main/scala/encry/consensus/PowLinearController.scala

```diff
@@ -10,8 +10,12 @@ object PowLinearController {
 
   private val chainParams = Constants.Chain
 
-  def getDifficulty(previousHeaders: Seq[(Int, EncryBlockHeader)]): NBits = {
-    if (previousHeaders.length == chainParams.RetargetingEpochsQty) {
+  val PrecisionConstant: Int = 1000000000
```

Comment:
 Magic number. 
I suggest to add this as bad practice to coding standard.

Reply:
This number appears only in special context of the private function above, so it would be not very strait forward to put it to global settings.